src/components/layout/CustomIntercom.js

```diff
@@ -64,4 +61,4 @@ function CustomIntercom(props) {
     }
 }
 
-export default CustomIntercom;
+export default withTranslation("common")(CustomIntercom);
```

Comment:
Is it possible to pull from multiple files?  Like if my component needs a string from `common` and also a string from `data`?

Reply:
Yes. use can use an array syntax here.